drivers/i2c/i2c_dw.c

```diff
@@ -63,7 +63,9 @@ static inline void i2c_dw_data_ask(const struct device *dev)
 	}
 
 	/* Get the FIFO depth that could be from 2 to 256 from HW spec */
+#if !DT_ANY_INST_PROP_STATUS_OKAY(aux_reg)
 	ic_comp_param_1.raw = read_comp_param_1(reg_base);
+#endif
```

Comment:
Shouldn't there be an #else for the DFSS case? And set the fifo size in the DT?

Reply:
This register is unrelated to FIFO, now moved comment to next line to avoid misleading. There is no such `COMP_PARAM_1` register in DFSS and we can safely ignore it, so no DFSS case.

Reply:
I know DFSS doesn't have these build config registers, but it has a fifo with a specific size. How do rx/tx_buffer_depth get the correct value for DFSS?

Reply:
Thanks for reminding me, now added `CONFIG_I2C_DW_FIFO_DEPTH`  for `AUX_REGS` case.